sentry-android-ndk/src/main/java/io/sentry/android/ndk/DebugImagesLoader.java

```diff
@@ -0,0 +1,79 @@
+package io.sentry.android.ndk;
+
+import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
+import io.sentry.android.core.IDebugImagesLoader;
+import io.sentry.android.core.SentryAndroidOptions;
+import io.sentry.protocol.DebugImage;
+import io.sentry.util.Objects;
+import java.util.Arrays;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.VisibleForTesting;
+
+final class DebugImagesLoader implements IDebugImagesLoader {
+
+  private final @NotNull SentryOptions options;
+
+  private final @NotNull IModuleListLoader moduleListLoader;
+
+  private static @Nullable List<DebugImage> debugImages;
+
+  /** we need to lock it because it could be called from different threads */
+  private static final @NotNull Object debugImagesLock = new Object();
+
+  DebugImagesLoader(
+      final @NotNull SentryAndroidOptions options,
+      final @NotNull IModuleListLoader moduleListLoader) {
+    this.options = Objects.requireNonNull(options, "The SentryAndroidOptions is required.");
```

Comment:
`l`
```suggestion
    this.options = Objects.requireNonNull(options, "The SentryAndroidOptions are required.");
```

Reply:
`SentryAndroidOptions` is the name of a class, so `is` is the correct term.